src/pages/events/summit-event-list-page.js

```diff
@@ -28,11 +28,11 @@ const fieldNames = [
     { columnKey: 'speakers', value: 'speakers' },
     { columnKey: 'created_by_fullname', value: 'created_by' },
     { columnKey: 'published_date', value: 'published' },
-    { columnKey: 'duration', value: 'duration' },
-    { columnKey: 'speaker_count', value: 'speaker_count' },
+    { columnKey: 'duration', value: 'duration', sortable: true },
+    { columnKey: 'speaker_count', value: 'speaker_count', sortable: true },
```

Comment:
the column should be called speakers_count check the filter counterpart
https://github.com/fntechgit/summit-admin/blob/2ef3e8b43cf21315a12bb480f0ac28a86d87c77f/src/actions/event-actions.js#L823